patroni/dcs/__init__.py

```diff
@@ -124,42 +100,14 @@ def iter_dcs_classes(
 
     :yields: a tuple containing the module ``name`` and the imported DCS class object.
```

Comment:
```suggestion
    :returns: an iterator of tuples, each containing the module ``name`` and the imported DCS class object.
```

I wonder if that describes the new situation more precisely? I mean, it's not generating values, but returning a generator now.